### PR TITLE
Add required bindings to the like count container. closes #562

### LINKED ISSUE #562: [FEATURE SUGGESTION] Do likes with click on heart, update count
Clicking on the heart icon next to the likes of a song should actually perform the like action. Currently this only works with the like button in the upper right corner. Also Increasing the like should immediately update the like count.

Here is an animation to make this more clear:

peek 2016-01-28 21-16

In case this is supposed to already work and it is a bug instead: I am running Soundnode 0.6.2 on Gnome 3.18, Arch Linux

### PR DESCRIPTION
**Pull request by vladjerca**
- clicking the count container will now trigger the like song event
- SCSS has been updated in order to set the active color after the song is liked

### CODE REVIEW

File: app/views/track/track.html [hunk 1]
```diff
@@ -9,7 +9,9 @@
                     <span class="playback_count">
                         <i class="fa fa-play"></i> {{track.playback_count | number}}
                     </span>
-                    <span class="likes_count">
+                    <span class="likes_count" ng-click="changeLike()"
```
Comment (weblancaster) [inline app/views/track/track.html]: why do we have `ng-click="changeLike()"` but isn't being used.

**Comment by weblancaster** [comment]
@vladjerca the heart state changes but theres a couple of things I noticed.. the number/counter doesn't change.. also there is a piece of code not in use `ng-click="changeLike()"`

Can you make the counter go up as well?

**Comment by vladjerca** [comment]
Yeah, I can take care of it.

**Comment by weblancaster** [comment]
@vladjerca let me know once you are done with this.

**Comment by vladjerca** [comment]
@weblancaster I've updated the directive definition and modified all affected html sections.

**Comment by weblancaster** [comment]
Thanks for the PR @vladjerca and apologies for the delay in review.

**Comment by vladjerca** [comment]
I love the app, only seems logical to give back how much I can.

**Pull request merged**